common/helpers.go

```diff
@@ -215,3 +215,9 @@ func TrimQuotes(s string) string {
 	}
 	return s
 }
+
+// isGojaValueExists returns true if a given value is not nil and exists
+// (defined and not null) in the goja runtime.
+func isGojaValueExists(v goja.Value) bool {
```

Comment:
`isGojaValueExists` is a bit awkwardly named, so maybe `isValidGojaValue` or `gojaValueExists`?